environment-approvals.yml

```diff
@@ -195,6 +195,7 @@ prod:
   - repo: https://github.com/hmcts/wa-task-monitor.git
   - repo: https://github.com/hmcts/wa-standalone-task-bpmn.git
   - repo: https://github.com/hmcts/civil-service.git
+  - repo: https://github.com/hmcts/civil-commons.git
```

Comment:
is this an actual service or a library?

Reply:
library

Reply:
no need to have this here then this file is for deploying to production